discovery-provider/alembic/wip_sql/migrate_agg_track.sql

```diff
@@ -0,0 +1,124 @@
+
+alter table aggregate_track alter column repost_count set default 0;
+alter table aggregate_track alter column save_count set default 0;
+
+
+WITH 
+aggregate_track_latest_blocknumber AS (
+  select last_checkpoint as blocknumber from indexing_checkpoints where tablename = 'aggregate_track'
+),
+new_track AS (
+    SELECT
+        t.track_id AS track_id
+    FROM
+        tracks t
+    WHERE
+        t.is_current IS TRUE
+        AND t.is_delete IS FALSE
+        AND t.blocknumber > (select blocknumber from aggregate_track_latest_blocknumber)
+    GROUP BY
+        t.track_id
+    UNION
+    ALL (
+        SELECT
+            r.repost_item_id AS track_id
+        FROM
+            reposts r
+        WHERE
+            r.is_current IS TRUE
+            AND r.repost_type = 'track'
+            AND r.is_delete IS FALSE
```

Comment:
for both reposts and saves, don't we want all the updates - delete and not delete after the blocknumber

Reply:
that's an good point... I copy pasted this verbatim from `index_aggregate_track` but it makes me think that decrements might not work correctly as written.